README.md

```diff
@@ -10,7 +10,7 @@ If you need to test functionality just run:
 
 ```bash
 docker pull lscr.io/linuxserver/d2-builder:latest && \
-docker run -d --rm -v /tmp/d2:/output -e PUID=1000 -e PGID=1000 lscr.io/linuxserver/d2-builder:latest mastodon:latest
+docker run -d --rm -v /tmp/d2:/output -e PUID=1000 -e PGID=1000 -e SVG=true lscr.io/linuxserver/d2-builder:latest mastodon:latest
```

Comment:
`SVG=true` is this still needed here?

Reply:
It's been removed from the readme-vars but I didn't rerun the builder